misk-hibernate/src/main/kotlin/misk/hibernate/HibernateModule.kt

```diff
@@ -56,6 +56,7 @@ class HibernateModule @JvmOverloads constructor(
   private val readerQualifier: KClass<out Annotation>?,
   readerConfig: DataSourceConfig?,
   val databasePool: DatabasePool = RealDatabasePool,
+  logLevelConfig: HibernateExceptionLogLevelConfig? = null,
```

Comment:
```suggestion
  logLevelConfig: HibernateExceptionLogLevelConfig = HibernateExceptionLogLevelConfig(),
```

Should this default to the default config instead of null? Then don't need the null check on L70